cmd/minikube/cmd/version.go

```diff
@@ -53,13 +54,14 @@ var versionCmd = &cobra.Command{
 			co := mustload.Running(ClusterFlagValue())
 			runner := co.CP.Runner
 			versionCMDS := map[string]*exec.Cmd{
-				"docker":     exec.Command("docker", "version", "--format={{.Client.Version}}"),
+				"docker":     exec.Command("docker", "--version"),
+				"dockerd":    exec.Command("dockerd", "--version"),
 				"containerd": exec.Command("containerd", "--version"),
-				"crio":       exec.Command("crio", "version"),
-				"podman":     exec.Command("sudo", "podman", "version"),
-				"crictl":     exec.Command("sudo", "crictl", "version"),
+				"crio":       exec.Command("crio", "--version"),
+				"podman":     exec.Command("sudo", "podman", "--version"),
+				"crictl":     exec.Command("sudo", "crictl", "--version"),
```

Comment:
It seems `--version` is not a valid flag for crictl
```
% sudo crictl --version
crictl version unknown

% sudo crictl version
Version:  0.1.0
RuntimeName:  docker
RuntimeVersion:  20.10.7
RuntimeApiVersion:  1.41.0
```

Reply:
It's a valid flag, just that the old binaries didn't have any version information. The newer binaries work:

```
$ sudo crictl --version
crictl version v1.21.0
```

The `crictl version` cmd doesn't output any information about `crictl`, only about the CRI and the API...

See #10610 for the bug